TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/cluster.py

```diff
@@ -177,8 +177,7 @@ def call_api(self, path, method: HttpMethod = HttpMethod.GET, data=None, headers
         return r
 
     def execute_benchmark_workload(self, workload: str,
-                                   workload_params='target_throughput:0.5,bulk_size:10,bulk_indexing_clients:1,'
-                                                   'search_clients:1'):
+                                   workload_params='bulk_size:10,bulk_indexing_clients:1'):
```

Comment:
`target_throughput` and `search_clients` are not supported in older workload types - forcing the version via `--distribution-version=1.0.0` prevented us from seeing these issues in the past.